src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -124,19 +125,22 @@ export class RgwBucketFormComponent implements OnInit {
     if (this.editing) {
       // Edit
       const idCtl = this.bucketForm.get('id');
-      this.rgwBucketService.update(bidCtl.value, idCtl.value, ownerCtl.value).subscribe(
-        () => {
-          this.notificationService.show(
-            NotificationType.success,
-            this.i18n('Updated Object Gateway bucket "{{bid}}"', { bid: bidCtl.value })
-          );
-          this.goToListView();
-        },
-        () => {
-          // Reset the 'Submit' button.
-          this.bucketForm.setErrors({ cdSubmitButton: true });
-        }
-      );
+      const versioningCtl = this.bucketForm.get('versioning');
+      this.rgwBucketService
+        .update(bidCtl.value, idCtl.value, ownerCtl.value, versioningCtl.value)
+        .subscribe(
+          () => {
+            this.notificationService.show(
+              NotificationType.success,
+              this.i18n('Updated Object Gateway bucket "{{bid}}"', { bid: bidCtl.value })
```

Comment:
I'm not sure if we should close the sentence with a dot in a notification. Is there any guideline for that?
```suggestion
              this.i18n('Updated Object Gateway bucket "{{bid}}".', { bid: bidCtl.value })
```

Reply:
Although the dot wasn't there even before this PR, I'll put it for consistency.